js/NewsChannel.js

```diff
@@ -0,0 +1,128 @@
+var bodyDiv;
+var rightPannel, leftPannel;
+const newsChannels =["All","Channel 1","Channel 2","Channel 3","Channel 4","Channel 5",
+"Channel 6", "Channel 7", "Channel 8", "Channel 9","Channel 10"];
+function GetChannelsInfo(){
+  CreateHeader();
+  bodyDiv = document.createElement('div');
+  bodyDiv.setAttribute('class','body');
+  leftPannel = document.createElement('div');
+  leftPannel.setAttribute('class','left-pane');
+  bodyDiv.appendChild(leftPannel);
+  rightPannel = document.createElement('div');
+  rightPannel.setAttribute('class','right-pane');
+  bodyDiv.appendChild(rightPannel);
+  document.body.appendChild(bodyDiv);
+  for(var index =1; index< newsChannels.length; index++)
+  {
+    LoadChannelInfo(index);
+  }
+    CreateCategoryinfo();
+  CreateFooter();
+}
+// Header Information
+function CreateHeader(){
+  var  headerInfo = document.createElement('div');
+  headerInfo.setAttribute('class','header');
+  let headerTag=document.createElement('h2');
+  headerTag.innerHTML="NEWSFEED";
+  headerTag.setAttribute('class','h2');
+  headerInfo.appendChild(headerTag);
+  var para= document.createElement("p");
+  let node=document.createTextNode("Yet Another News Feed");
+  para.setAttribute('class','description');
+  para.appendChild(node);
+  headerInfo.appendChild(para) ;
+  document.body.appendChild(headerInfo);
+  return headerInfo;
```

Comment:
Why do we need this return statement when you are already adding to body?

Reply:
It was added by mistake. Removed it now